gradle/libs.versions.toml

```diff
@@ -255,9 +256,7 @@ managed-jna = { module = "net.java.dev.jna:jna", version.ref = "managed-jna" }
 
 managed-jsr305 = { module = "com.google.code.findbugs:jsr305", version.ref = "managed-jsr305" }
 
-managed-kafka = { module = "org.apache.kafka:kafka-clients", version.ref = "managed-kafka" }
 managed-kafka212 = { module = "org.apache.kafka:kafka_2.12", version.ref = "managed-kafka" }
```

Comment:
why do we have this? Do we know?

Reply:
I have no idea...  I originally deleted it, but I guess someone may be consuming it as a dependency, so I added it back in...

@graemerocher Do you recall where this is required?  I can find it in 2018

My vote would be to delete it for 3.5.0, and see who squeaks

Reply:
I don't think that `kafka212` is a great name either. Obviously encoding the version in the name of the alias kind of defeats the purpose. Maybe this should be reworked for 4.0.